Calendar/Get-CalendarDiagnosticObjectsSummary.ps1

```diff
@@ -257,6 +256,66 @@ function GetMailbox {
     }
 }
 
+<#
+.SYNOPSIS
+Checks if a set of Calendar Logs is from the Organizer.
+#>
+function SetIsOrganizer {
+    param(
+        $CalLogs
+    )
+    $IsOrganizer = $false
+
+    foreach ($CalLog in $CalLogs) {
+        if ($CalendarItemTypes.($CalLog.ItemClass) -eq "IpmAppointment" -and
+            $CalLog.ExternalSharingMasterId -eq "NotFound" -and
+            ($CalLog.ResponseType -eq "1" -or $CalLogs.ResponseType -eq "Organizer")) {
+            $IsOrganizer = "True"
```

Comment:
This should be a boolean of `$true`, not a string if we are originally setting `$IsOrganizer = $false`

Reply:
Done